magento/pwa-studio
# [feature]: Replace ProductQuantity on Product Page with QuantityFields stepper

## Description

The quantity selector on PDP and cart don't currently match either in the way they look or in the way they behave.

The cart stepper is the latest design, and the quantity selector on PDP should be updated to match this.

## Related Issue





Closes #2679.

## Acceptance 



### Verification Stakeholders

@tjwiebell 
### Specification



### Verification Steps


1. Go to the PDP.
2. Verify the quantity field is QuantityFields.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
